federation/index.md

```diff
@@ -88,6 +88,45 @@ only either ARD processes **or** "standard" processes can be used in one process
 is not yet supported. One option to nevertheless achieve a combination of process types is to run ARD on Level-1 data,
 save the results of the job, and then in a second job load results and perform additional computations.
 
+### Enforce back-end selection for common collections
+
+Some collections are provided by multiple underlying back-ends,
+possibly with differences in spatial or temporal coverage.
+This is exposed in the collection metadata with `federation:backends` summary, e.g.:
+
+```json
+{
+    "id": "WATER_BODIES",
+    "type": "Collection",
+    ...
+    "summaries": {
+        "federation:backends": ["vito", "sentinelhub"],
+        ...
+    }
+}
+```
+
+When a user submits a processing request,
+the federated platform will try, by default, to automatically determine
+which underlying back-end is best choice for the actual processing,
+based for example on the requested spatial extent.
+
+You can however also enforce the selection of a certain back-end
+by using the metadata property filtering feature
+of the `load_collection` process.
+For example, with the Python client, to enforce the selection of
+the "sentinelhub" back-end:
+
+```python
```

Comment:
It would make sense here to also have an example in R.
If I remember it correctly it should work as follows:

```r
cube = p$load_collection(
    id = "WATER_BODIES",
    ...
    properties = list("federation:backends" = function(x) x == "sentinelhub")
)
```
Here's an example how to use the CodeSwitcher component: https://github.com/openEOPlatform/documentation/blob/main/usecases/no2-monitoring/index.md?plain=1#L47

JS doesn't have a good implementation for property filtering yet. :-(

Reply:
added R code example

Reply:
You need empty lines between the code block and the template tags, see https://github.com/openEOPlatform/documentation/blob/main/usecases/no2-monitoring/index.md?plain=1#L47 for an example. This is unfortunately some kind of limitation of the CodeSwitcher implementation or VueJS.